profile/profile.md

```diff
@@ -0,0 +1,56 @@
+# Haotian Zhu . Jarvis Consulting
+
+I am Haotian Zhu, who graduated from the University of Alberta in 2020.  I am a versatile software application developer with a passion for learning and delivering on applications with a high-quality standard. I was working on campus as a full-stack developer. My primary responsibility was to develop a restful back-end API and do integrate tests with the front-end. I currently work on Jarvis as a software developer. I have experience using Django, Nodejs, and SpringBoot for the back-end; meanwhile, I am familiar with Reactjs, MongoDB and PostgreSQL. I am a team-oriented person, and I am also a problem solver. I enjoy coding challenges and learning algorithms and data structures.  Currently, I am looking for a great team to provide my skills and learn new tech skills.
+
+## Skills
+
+**Proficient:** Java, Python, JavaScript, Django, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** HTML5, CSS, React, Postman, Docker, CI/CD
+
+**Familiar:** Azure Cloud/Heroku Cloud, Express.js, Angular7, MongoDB, Jquery
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_HaotianZhu](https://github.com/jarviscanada/jarvis_data_eng_HaotianZhu)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HaotianZhu/tree/master/linux)]: Coded bash scripts to generate PSQL database quickly via docker commands. Such PSQL database deployed into Cloud will record hardware specifications of multiple Linux servers/nodes. Developed SQL script to automatically insert Linux server hardware information into the database. It allows the Jarvis Linux Cluster Administration team to track and manage a Linux cluster of 10 nodes/server.
```

Comment:
~Coded bash scripts to generate PSQL database~

Developed bash scripts to deploy PostgreSQL database with docker.

Reply:
Developed SQL script to automatically insert Linux server hardware information into the database

Developed a bash agent program that automatically parses hardware specifications and real-time hard usage data, which is persisted into the PostgreSQL database.

Reply:
It allows the Jarvis Linux Cluster Administration team to track and manage a Linux cluster ~of 10 nodes/server.~